src/tests/cli.cpp

```diff
@@ -471,3 +471,27 @@ test_cli_rnpkeys(void **state)
     ret = call_rnp("rnpkeys", "--homedir", KEYS "/1", "--list-keys", "zzzzzzzz", NULL);
     assert_int_not_equal(ret, 0);
 }
+
+void
+test_cli_redumper(void **state)
+{
+    rnp_test_state_t *rstate = (rnp_test_state_t *) *state;
+    char *            redumper_path =
+      rnp_compose_path(rstate->original_dir, "../apps/packet-dumper/redumper", NULL);
+    char cmd[512] = {0};
+    int  chnum;
+    int  status;
+    /* call redumper's help */
+    chnum = snprintf(cmd, sizeof(cmd), "%s -h", redumper_path);
+    assert_true(chnum < (int) sizeof(cmd));
+    status = system(cmd);
+    assert_true(WIFEXITED(status));
```

Comment:
It looks like you'll need to add `#include <sys/wait.h>` for `WIFEXITED`etc on FreeBSD. That should be OK for all 3 supported platforms.

Reply:
@dewyatt Thanks, now it should be ok. Worked on macOS without include.